src/AltinnCore/Designer/Dockerfile

```diff
@@ -19,8 +19,8 @@ FROM altinn-studio-dashboard:latest AS altinn-studio-dashboard
 
 FROM altinn-runtime-app:latest AS generate-runtime-app
 
-#dotnet:2.1.300-sdk
-FROM microsoft/dotnet@sha256:d1ad61421f637a4fe6443f2ec204cca9fe10bf833c31adc6ce70a4f66406375e AS final
+#dotnet:2.2-aspnetcore-runtime
```

Comment:
why is there a difference of "aspnetcore-runtime" here?

Reply:
Because we need to use 2.2's runtime when project is running on 2.2

Reply:
ya, i just wondered as the 2.1.300 is not having an extension as aspnetcore-runtime. So is it just that 2.2 has tagged with runtime which is not the case with previous versions?

Reply:
I think the previous comment is just outdated, so the sha before was the sha reference to the runtime 2.1